### DISCUSSION TITLE
Nothing happening on click for wallets other than metamask

### DISCUSSION THREAD

**Discussion by pcdkd** [rainbow-me/rainbowkit#1118] [Feedback]
See screen record link below

https://www.dropbox.com/s/3jaafwymw0ovu8b/Screen%20Recording%202023-03-27%20at%2007.12.34.mov?dl=0

Link to repo here

https://github.com/pcdkd/polygononchain-minter

**Comment by Hussainzz** [comment]
https://www.youtube.com/watch?v=TLpPN9mN_OM&t=226s

**Reply by pcdkd** [reply]
What are you trying to highlight here? I'll need to install via custom wallet method? Why wouldn't default wallet method be working here?
